client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -81,38 +70,58 @@ const InvestigationForm: React.FC = (): JSX.Element => {
             setSymptomsStartDate, setExposureDate, setHasSymptoms, setEndInvestigationDate]
     );
 
-    const { currentTab, setCurrentTab, confirmFinishInvestigation, handleSwitchTab, saveCurrentTab, isButtonDisabled } = useInvestigationForm({ clinicalDetailsVariables, personalInfoData, exposuresAndFlightsVariables });
+    const { confirmFinishInvestigation } = useInvestigationForm({ clinicalDetailsVariables, exposuresAndFlightsVariables });
+    const {
+        currentTab,
+        moveToNextTab,
+        setNextTab
+    } = useTabManagement();
+
+    const isInvestigationValid = () => {
+        return !(formsValidations.some((formValidation) => !formValidation));
+    }
+
+    const handleClick = () => {
+        if(currentTab === LAST_TAB_ID) { 
+            if(isInvestigationValid()) {
+                confirmFinishInvestigation(epidemiologyNumber);
+            } else {
+                Swal.fire({
```

Comment:
add font to swal

Reply:
![image](https://user-images.githubusercontent.com/24894463/95018059-c5819700-0665-11eb-8510-4e95e4b36320.png)